Formula/cmake.rb

```diff
@@ -32,6 +34,10 @@ class Cmake < Formula
   # For the GUI application please instead use `brew install --cask cmake`.
 
   def install
+    on_linux do
+      ENV.cxx11
```

Comment:
Isn't this a no-op nowadays?

Reply:
Maybe unrelated, but GCC is still at version 5 on Linux (Migration to GCC 10 is planned)

Reply:
Err yes, this is one small "detail" I forgot to tell you @fxcoudert. 
We still build stuff with system gcc 5, because that's a LTS version on ubuntu 16.04. This might change next year when 16.04 is gone, then we will migrate to gcc 7.

Another solution to handle this is to finalise the migration of our brewed gcc formula in linuxbrew-core to be gcc 10; and use gcc 10 to build cmake. But for this we need self-hosted CI, and this will take some time until I have set it up.

@SeekingMeaning did you try the build without that flag?

Reply:
Works without flag with GCC 5 on Ubuntu 20.04